client/js/Core/main.js

```diff
@@ -1,70 +1,75 @@
-var scHelper = null;
-var scKeynodes = null;
+const scHelper = null;
+const scKeynodes = null;
 const currentYear = new Date().getFullYear();
 
-SCWeb.core.Main = {
+function ScClientCreate() {
+    let res, rej;
+    let serverHost = "ws://localhost:8090/ws_json";
```

Comment:
Wait a second. We can't presume that we're always having an instance of sc-machine running on the machine that the frontend loaded on. We need a dynamic mechanism to generate this link, it cannot be hardcoded